SimpleInventoryManagementSystem/InputLogger.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using SimpleInventoryManagementSystem;
+
+
+namespace SimpleInventoryManagementSystem;
+
+public class InputLogger
+{
+    ProductFactory _productFactory;
+
+    public InputLogger()
+    {
+        _productFactory = new ProductFactory();
+    }
```

Comment:
Missing access modifier, lets make this `readonly` and better use the Inventory to avoid the `ProductFactory`.

(Optional) We can also remove the constructor.

```suggestion
    private readonly Inventory _inventoryManager = new Inventory();
```